### PR TITLE
Playlist page improvements

### LINKED ISSUE #2255: [Bug]: Some elements in the dropdown on the playlist page are inaccessible
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a bug report that matches the one I want to file, without success.

### Describe the bug

1. Go to https://youtube.com/playlist?list=PLTariUrSxayvzHkCnEiX5-NSeFgLTqzHE
2.

https://user-images.githubusercontent.com/73130443/169654753-45636bd2-4ce0-49ca-8972-971bcf73eb0b.mp4

 

### Expected Behavior

I'm always able to access the dropdown box

### Issue Labels

accessibility issue, usability issue

### FreeTube Version

https://github.com/FreeTubeApp/FreeTube/commit/d34720aeb3f5c5d7c9a80d9d26bc55c1f01e07fb

### Operating System Version

Win10 21H1

### Installation Method (If applicable)

.exe

### Last Known Working FreeTube Version (If Any)

_No response_

### Primary API used

_No response_

### Additional Information

_No response_

### Nightly Build

- [X] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

### PR DESCRIPTION
**Pull request by vallode**
---
Playlist page improvements
---

**Pull Request Type**
Please select what type of pull request this is:
- [x] Bugfix

**Related issue**
Closes #2255

**Description**
Small love and care to the playlist templates:

* Add thumbnail hover visual on videos
* Fixes issue with overflow not being visible on the playlist description
* Fixes issue where playlist description whitespaces would not be rendered
* Changes the channel icon to be a bit smaller to save space
* Changes a few styles to the playlist stats

**Screenshots (if appropriate)**
Please add before and after screenshots if there is a visible change.

Before:
image

After:
image

**Testing (for code that is not small enough to be easily understandable)**
Just played around with playlists and smaller views.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] 10
 - OS Version: [e.g. 22] Debian
 - FreeTube version: [e.g. 0.8] upstream/development

### CODE REVIEW

**Review by absidue — CHANGES_REQUESTED** [review]
Please fix the missing new lines and the share menu getting cut off.

Playlist in the screenshots: https://www.youtube.com/playlist?list=PLDIoUOhQQPlXr63I_vwF9GD8sAKh77dWU

Share menu in development:
development-share

Normal with this PR:
this-pr-normal

Hovering over the share menu with this PR:
this-pr-share

**Review by absidue — DISMISSED** [review]
Looks good now. Thanks.

File: src/renderer/components/playlist-info/playlist-info.vue [hunk 1]
```diff
@@ -8,21 +8,25 @@
...
+      </p>
+    </div>
+
     <p
       class="playlistDescription"
-    >
-      {{ description }}
-    </p>
+      v-text="description"
+    ></p>
```
Comment (ChunkyProgrammer) [inline src/renderer/components/playlist-info/playlist-info.vue:28]: lint issue
```suggestion
    >
```

**Review by PikachuEXE — APPROVED** [review]

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]
LGTM!

**Pull request merged**